crates/uv/src/commands/python/pin.rs

```diff
@@ -67,18 +67,19 @@ pub(crate) async fn pin(
     } else {
         request.to_canonical_string()
     };
-
-    debug!("Using pin `{}`", output);
     let version_file = PathBuf::from(PYTHON_VERSION_FILENAME);
-    let exists = version_file.exists();
 
-    debug!("Writing pin to {}", version_file.user_display());
+    debug!(
+        "Writing pinned Python version `{}` to {}",
+        output,
+        version_file.user_display()
+    );
     fs_err::write(&version_file, format!("{output}\n"))?;
-    if exists {
-        writeln!(printer.stdout(), "Replaced existing pin with `{output}`")?;
```

Comment:
I opted to remove this, I didn't find it necessary, but not a strong opinion. If we _do_ show it, I feel like it should tell you what we updated the pin _from_. That adds some complexity because it could be multiple values, right?

Reply:
I saw Cargo use "Replaced" in its CLI and I liked it. I would like to show what we change from, that sounds nice. If it's multiple values, I think we can just take the first one which is effectively the "pinned" version.

Reply:
Ok

Reply:
The utility to retrieve the version request from files will retrieve the first line for you already